src/options/filter.rs

```diff
@@ -294,7 +294,7 @@ mod test {
     mod ignore_patterns {
         use super::*;
         use std::iter::FromIterator;
-        use glob;
+        
```

Comment:
You can remove the line, there should only be one line between the `use` block and the next block.